### ISSUE TITLE
Update all `secret: ` values in `.travis.yml`

### ISSUE THREAD

**Issue by webknjaz** [aio-libs/aiohttp-debugtoolbar#212]
Do like this:
```
REPO_NAME={{ YOUR_REPO_NAME }}
travis encrypt -r "aio-libs/${REPO_NAME}" --api-endpoint 'https://api.travis-ci.com/'
```

Ref: https://github.com/orgs/aio-libs/teams/admins/discussions/9

**Comment by webknjaz** [comment]
```yaml
secure: "n4dE3kWtpvgeVzFkWynvRfZ6zhdrVyucs/cMqYPm0QRlwV14Iv5MLXJQ3DRPXiKgVs0vQKPzJFRPvFrYh21ugCmX+E/ToRS6O1BkDOEnf2yuYh/sY6U8+Sk945PAmMVfj9BjRapbu7kSGq0YtAcfdo3PAvBB5fHCjA//E8jHNxRjMCXpsjSiJTvL+OojQO0HZ7fAsfJQERt6hlIzxujNvG3UuXWHxvEUPiNJRjy/gHh4OS7SK4lnCfB7l/LTd5JIO1nJM8BDLfPyaurUHxRVzrP0JZrcl72wLofbPuniRaxlQMcDrFyBWmThoQj42RbHkaCtDBJmG8bxCzo0q2KUZuAUvgBvhIXqI/sCk0+CKpJFAVRTB6DH60aqsKEHu+zLIQMJATvCUhh7vOCov2V2Cok7kYf7W87E+iE9p7gPax+UuhzsYptygexv91EFxRcNq9kGbLYoN2RION5wwyEN25U3mYZcPjWpeW1C3655fcWQm3GMB4+pbNWAeycbL29FfXErAzXJUeCxhN1ytp7u0QbiuWoZDcW+RCJQMq2+5xQhR2JBMqElac3Q5jAI38TkXAsXY82ddRaYbWyO8rHq4tsXfX1sDhPTuqMz1tSgOyrAqxR/3oaYUplV2A2nexrjj6VNihLll/BoSNCVCO/NZEB3E0Ny3JuriiyvFB6v5wo="
```

**Comment by webknjaz** [comment]
user: aio-libs-bot

**Comment by asvetlov** [comment]
Fixed with 0573ba8
